aws-connect

```diff
@@ -208,6 +220,55 @@ elif [ "${action}" == "tunnel" ]; then
     --parameters "{\"portNumber\":[\"22\"],\"localPortNumber\":[\"${port}\"]}" \
     --region "${aws_region}" \
     --profile "${aws_profile}"
+elif [ "${action}" == "document" ]; then
+
+  if [ -z "${github_token_location}" ]
+  then
+    echo "No github token"
+    document_parameters="parameters='${document_parameters}'"
+  else
+    document_parameters="githubTokenLocation='${github_token_location}',parameters='${document_parameters}'"
+    echo "Github token found"
+  fi
+
+  echo "Running ${document_name} on instance ${instance_id}"
+  
+  sh_command_id=$( aws ssm send-command \
+    --instance-ids ${instance_id} \
+    --document-name "${document_name}" \
+    --comment "Document run using aws-connect" \
+    --output text \
+    --parameters "${document_parameters}" \
+    --region ${aws_region} \
+    --profile ${aws_profile} \
+    --query "Command.CommandId") 
+
+  output=$(aws ssm list-command-invocations \
+    --command-id ${sh_command_id} \
+    --details)
+
+  status=$(echo $output | jq -r '.[]' | jq -r '.[]' | jq -r '.Status')
+
+  # wait for the command to finish running before getting the output
+  while [ "$status" == "InProgress" ]
+  do
+    echo "Command still running..."
+    sleep 5
+
+    output=$(aws ssm list-command-invocations \
+      --command-id ${sh_command_id} \
+      --details)
+
+    status=$(echo $output | jq -r '.[]' | jq -r '.[]' | jq -r '.Status' )
+  done
+
+  if [ "$status" == "Failed" ]
+  then
+    echo "Command Failed!"
+  fi
+
+  echo $output | jq -r '.[]' | jq -r '.[]' | jq -r '.CommandPlugins' | jq -r '.[1]' | jq -r '.Output'
```

Comment:
So I believe the output gets truncated if it's too big for the output field. In that case, it gets written to S3.  Again though here, don't use jq, use query

https://docs.aws.amazon.com/cli/latest/reference/ssm/send-command.html

```
[--output-s3-region <value>]
[--output-s3-bucket-name <value>]
[--output-s3-key-prefix <value>]
```

You'll need to test running a script that produces larger output and see what you get back in output.  I think we may need to pass in an S3 bucket to throw the results into and then return the output location to the user.  The prefix we can auto-generate (say based on the command ID) but if we don't pass a bucket, we'll lose the output